tests/test_cli_entry.py

```
@@ -0,0 +1,60 @@
+import sys
+from pathlib import Path
+
+
+def test_odin_cli_import():
+    """Test that odin_cli.py can import correctly."""
+    # Add project root to path
+    project_root = Path(__file__).parent.parent
+    sys.path.insert(0, str(project_root))
+    
+    try:
+        from odin.cli import cli_main, main_research
```

Ruff (0.11.9)

`odin.cli.cli_main` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)